contracts/axelarnet-gateway/tests/execute.rs

```diff
@@ -309,6 +313,40 @@ fn contract_call_returns_correct_message() {
     goldie::assert_json!(msg)
 }
 
+#[test]
+fn contract_call_with_token_returns_correct_message() {
+    let tx_hash: [u8; 32] =
+        hex::decode("b795e27bcb71c3dfd108c18e031ec966e37c7c95927c2a9fd88ec573ee690c2c")
+            .unwrap()
+            .try_into()
+            .unwrap();
```

Comment:
```suggestion
    let tx_hash: [u8; 32] = [1; 32];
```
keep it simple since value doesn't matter?